### DISCUSSION TITLE
where to find documentation on 'evaluate-sequence' option?

### DISCUSSION THREAD

**Discussion by LateJunction** [ImageMagick/ImageMagick#6106] [Help]
### ImageMagick version

7.1.0-62

### Operating system, version and so on

Linux Ubuntu 22.04/Mint 21.1, Cinnamon

### Description

In practice I'm new to Imagemagick and a proven slow-learner. I am struggling to find the correct syntax for the '-evalute-sequence' option of magick. Where should I be looking for the documentation?  In addition I was trying to use the 'convert' command (with the '-evaluate-sequence' option) but this command seems to no longer be required. Is that correct?

**Comment by snibgo** [comment]
You have IM v7. `convert` would use the old (very old) v6 syntax. I suggest using just `magick` instead (and not `magick convert`).

For `-evaluate-sequence`, see https://imagemagick.org/script/command-line-options.php#evaluate-sequence . The command "`magick -list evaluate`" will list the available operators. The operators are described at https://imagemagick.org/script/command-line-options.php#evaluate

**Reply by LateJunction** [reply]
Thanks for this - it's the pointer I needed. What I find fascinating - and frustrating - is that there is no mention of this /script/ web-page at imagemagick.org so I would never have found it. Like so much else in the Linux world, the documentation is written by talented people who are subject matter experts and write for people like themselves - who, by definition, don't need it. But people like me end up in maelstrom of confusion - let alone having to fight through the madness of 'dependency deadly embrace' and 'root-and-branch' changes to command syntax.. But, never give up ....

**Comment by urban-warrior** [comment]
Enter "evaluate-sequence" in the search box at the top of each ImageMagick page.  The top search result is the most relevant page.  Perhaps less intuitive is you need to click on the drop down list of options and select `evaluate-sequence` from the list.

Note, ImageMagick is open source and all contributors are unpaid volunteers.  Anyone in the community can contribute to improve ImageMagick by, for example, improving the web pages (such as contributing a write-up of the `-evaluate-sequence` operator).  Just create a pull request @ https://github.com/ImageMagick/Website.
